src/v/config/configuration.cc

```diff
@@ -80,6 +80,28 @@ configuration::configuration()
       "Duration after which inactive readers will be evicted from cache",
       {.visibility = visibility::tunable},
       30s)
+  , log_segment_ms(
+      *this,
+      "log_segment_ms",
+      "Default log segment lifetime in ms for topics which do not set "
+      "segment.ms",
+      {.needs_restart = needs_restart::no, .visibility = visibility::user},
+      std::nullopt,
+      {.min = 1s})
```

Comment:
too low, min should be in minute+. @mattschumpert

Reply:
I'd be fine with a 60s minimum here too, as an extra measure to protect self hosted folks from themselves.  It'll just make our tests a bit slower when we can't configure it any lower than that.

log_segment_ms_min is the important guardrail (the one that's already set to 60s).  We will need to think separately about what we set that to in managed systems.